Mathlib/GroupTheory/Subsemigroup/Centralizer.lean

```diff
@@ -122,6 +122,17 @@ theorem centralizer_subset [Mul M] (h : S ⊆ T) : centralizer T ⊆ centralizer
 #align set.centralizer_subset Set.centralizer_subset
 #align set.add_centralizer_subset Set.add_centralizer_subset
 
+@[to_additive add_center_subset_add_centralizer]
```

Comment:
I agree with what you said on Xena, it should be
```suggestion
@[to_additive addCenter_subset_addCentralizer]
```

Reply:
I'll fix these in this PR and I'll fix the rest in a subsequent PR